integration/kubeconfig/kubeconfig_test.go

```diff
@@ -25,24 +26,46 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// Test_KubeconfigHasExec kubeconfig command should use exec instead of token for the user auth
+// Test_KubeconfigHasExec kubeconfig command should use exec instead of token for the user auth if feature flag enabled
 func Test_KubeconfigHasExec(t *testing.T) {
-	t.Parallel()
+	tests := []struct {
+		name           string
+		useStaticToken bool
+	}{
+		{
+			name:           "enabling static token feature flag",
+			useStaticToken: true,
+		},
+		{
+			name:           "disabling static token feature flag",
+			useStaticToken: false,
+		},
+	}
 
 	oktetoPath, err := integration.GetOktetoPath()
 	require.NoError(t, err)
 
 	home := t.TempDir()
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Setenv(kubeconfig.OktetoUseStaticKubetokenEnvVar, strconv.FormatBool(tt.useStaticToken))
 
-	err = commands.RunOktetoKubeconfig(oktetoPath, home)
-	require.NoError(t, err)
+			err = commands.RunOktetoKubeconfig(oktetoPath, home)
```

Comment:
Can we also test that kubetoken output is json parseable?

Reply:
I think we already have that test -> https://github.com/okteto/okteto/blob/master/integration/kubetoken/kubetoken_test.go#L42. Is that what you mean?